### PR TITLE
coova-chilli: Update to 1.4

### PR DESCRIPTION
**Pull request by neheb**
Switch to codeload for simplicity.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer @teslamint 
Compile tested: ipq806x

### CODE REVIEW

**Comment by teslamint** [comment]
```
In file included from system.h:133:0,
                 from chilli.h:24,
                 from chilli.c:21:
/tmp/tmp.SAPgnCltUM/staging_dir/toolchain-mips_24kc_gcc-7.3.0_musl/include/sys/sysinfo.h:10:8: error: redefinition of 'struct sysinfo'
 struct sysinfo {
        ^~~~~~~
In file included from /tmp/tmp.SAPgnCltUM/staging_dir/toolchain-mips_24kc_gcc-7.3.0_musl/include/linux/kernel.h:4:0,
                 from /tmp/tmp.SAPgnCltUM/staging_dir/toolchain-mips_24kc_gcc-7.3.0_musl/include/linux/netlink.h:4,
                 from system.h:126,
                 from chilli.h:24,
                 from chilli.c:21:
/tmp/tmp.SAPgnCltUM/staging_dir/toolchain-mips_24kc_gcc-7.3.0_musl/include/linux/sysinfo.h:7:8: note: originally defined here
 struct sysinfo {
        ^~~~~~~
make[6]: *** [chilli.lo] Error 1
make[6]: Leaving directory `/tmp/tmp.SAPgnCltUM/build_dir/target-mips_24kc_musl/coova-chilli-1.4/src'
```

I think `net/coova-chilli/patches/100-fix-sysinfo-redeclaration.patch` still needs for this problem.

File: net/coova-chilli/patches/100-fix_compile_kmod.patch [hunk 1]
```diff
@@ -1,10 +1,10 @@
 --- a/src/linux/Makefile
 +++ b/src/linux/Makefile
-@@ -21,11 +21,11 @@ lib%.so: lib%.o
+@@ -22,11 +22,11 @@ lib%.so: lib%.o
  	$(CC) $(CFLAGS) -shared -o $@ $^;
  
  lib%.o: lib%.c
--	$(CC) $(CFLAGS) -fPIC -O2 -Wall -I${KERNEL_DIR}/include -D_INIT=lib$*_init -c -o $@ $<;
```
Comment (teslamint) [inline net/coova-chilli/patches/100-fix_compile_kmod.patch]: Is kernel module builds without this?

**Comment by neheb** [comment]
I removed it since it didn't seem to be valid anymore. I will check again.

**Comment by neheb** [comment]
It was actually patch 400. Applied a simpler version.

**Comment by teslamint** [comment]
My WNDR3700v2's WAN is broken, but coova-chilli is running without crash, at least.

**Pull request merged**